partner_address_two_lines/models/res_partner.py

```diff
@@ -1,16 +1,40 @@
-# Copyright 2020 Camptocamp SA
+# Copyright 2024 Camptocamp SA
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 
-from odoo import models
+from odoo import api, models
 
 
 class ResPartner(models.Model):
     _inherit = "res.partner"
 
-    def _get_contact_name(self, partner, name):
+    def _get_complete_name(self):
```

Comment:
Traceback (most recent call last):
  File "/home/chien/code/oca/partner-contact/17.0/partner_address_two_lines/tests/test_res_partner.py", line 35, in test_get_name
    self.child_partner_name.display_name, "Test Company Name\nTest Partner Name"
  File "/home/chien/code/odoo/odoo/17.0/odoo/fields.py", line 1206, in __get__
    self.compute_value(recs)
  File "/home/chien/code/odoo/odoo/17.0/odoo/fields.py", line 1388, in compute_value
    records._compute_field_value(self)
  File "/home/chien/code/odoo/odoo/17.0/odoo/models.py", line 4858, in _compute_field_value
    fields.determine(field.compute, self)
  File "/home/chien/code/odoo/odoo/17.0/odoo/fields.py", line 101, in determine
    return needle(*args)
  File "/home/chien/code/oca/partner-contact/17.0/partner_address_two_lines/models/res_partner.py", line 39, in _compute_display_name
    name = rec._get_complete_name()
  File "/home/chien/code/oca/partner-contact/17.0/partner_address_two_lines/models/res_partner.py", line 11, in _get_complete_name
    res = super()._get_complete_name()
AttributeError: 'super' object has no attribute '_get_complete_name'